pulseeq/constants.py.in

```diff
@@ -1,7 +1,7 @@
 import os
 
-if 'PULSE_CONFIG' in os.environ:
-    CONFIG_DIR = os.path.dirname(os.getenv('PULSE_CONFIG'))
+if 'PULSE_CONFIG_PATH' in os.environ:
+    CONFIG_DIR = os.getenv('PULSE_CONFIG')
```

Comment:
What about this? Shouldn't it be `PULSE_CONFIG_PATH`.

Reply:
Nice catch, you're correct. I just fixed this.